proto/autokitteh/triggers/v1/trigger.proto

```diff
@@ -9,7 +9,7 @@ import "buf/validate/validate.proto";
 message Trigger {
   string trigger_id = 1;
   string name = 2;
-  string connection_id = 3;
+  string connection_id = 3; // could be empty
```

Comment:
you don't really this comment
all fields are optional in proto by default

Reply:
you are right. But I want to provide addition info why there is no validation check